### PR TITLE
ui: shadcn svelte -> tooltips + sonner

### LINKED ISSUE #259: feat: tooltips text for icons
**Is your feature request related to a problem? Please describe.**
Please add tooltips to icons which do not have any text attached to them

**Describe the solution you'd like**
A tiny description of the what the icon does should appear on hover

### PR DESCRIPTION
**Pull request by jannikstdl**
fix #259

[shadcn-svelte](https://www.shadcn-svelte.com/)

https://github.com/open-webui/open-webui/assets/69747628/836db88c-8154-4c3a-8469-c4ab7383eb25

**Edit** 
Also implemented the Sonner/Toaster from svelte-shadcn (they use [svelte-sonner](https://svelte-sonner.vercel.app/)).
Replaced it with svelte-french-toast since i found it more clean and it stacks:

https://github.com/open-webui/open-webui/assets/69747628/104efc2f-b1f8-4813-be53-f3647fc69aad

Maybe in future we could replace the model selector with [shadcns combobox](https://www.shadcn-svelte.com/docs/components/combobox). Somebody mentioned this on Discord that the native selector is inconsistent in design.

### CODE REVIEW

File: src/lib/components/chat/MessageInput.svelte [hunk 1]
```diff
@@ -633,25 +636,34 @@
...
+									<Tooltip.Trigger>
+										<button
+											class="  text-gray-600 dark:text-gray-200 transition rounded-lg p-1 ml-1"
+											type="button"
+											on:click={() => {
+												filesInputElement.click();
+											}}
+										>
+											<svg
+												xmlns="http://www.w3.org/2000/svg"
```
Comment (tjbck) [inline src/lib/components/chat/MessageInput.svelte:650]: We changed our icon here :)

File: package.json [hunk 2]
```diff
@@ -1,56 +1,62 @@
...
-		"katex": "^0.16.9",
-		"marked": "^9.1.0",
-		"svelte-french-toast": "^1.2.0",
-		"tippy.js": "^6.3.7",
-		"uuid": "^9.0.1"
-	}
-}
\ No newline at end of file
+  "name": "open-webui",
+  "version": "0.1.103",
```
Comment (tjbck) [inline package.json:3]: Should be "0.1.106" here!

File: CHANGELOG.md [hunk 3]
```diff
@@ -10,10 +10,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - **📄 Document Selection**: Now you can select and delete multiple documents at once for easier management.
+- **💡 Tooltips Added**: Tooltips have now been implemented on UI elements for enhanced guidance.
```
Comment (tjbck) [inline CHANGELOG.md:13]: Same here, should be under "0.1.106".

**Comment by tjbck** [comment]
Thanks for the PR! [svelte-sonner](https://svelte-sonner.vercel.app/) I think is great and looks better than the `svelte-toast` but I'm still debating if it's a good idea to include external UI libraries to our project. If Tooltip is all we need from the library, it's probably best if we implement one our own, I should be able to take a deep dive into this and create one later this week, how does that sound?

**Comment by jannikstdl** [comment]
> Thanks for the PR! [svelte-sonner](https://svelte-sonner.vercel.app/) I think is great and looks better than the `svelte-toast`

Should I cerate a PR with only svelte-sonner?

**Comment by tjbck** [comment]
@jannikstdl yep, that would expedite the PR merge!

**Comment by jannikstdl** [comment]
Splitted in #986 and #982

**Pull request closed without merging**